plugins/installer/webinstaller/webinstaller.script.php

```diff
@@ -21,8 +21,12 @@ class plginstallerwebinstallerInstallerScript
         public function postflight($route, $adapter)
         {
             $db = JFactory::getDbo();
-            $query = 'UPDATE ' . $db->quoteName('#__extensions') . ' SET ' . $db->quoteName('enabled') . ' = 1 WHERE ' . $db->quoteName('type') . ' = ' . $db->quote('plugin') . ' AND ' . $db->quoteName('element') . ' = ' . $db->quote('webinstaller');
+            $query = $db->getQuery(true);
+            $query->update($db->quoteName('#__extensions'))
+                ->set($db->quoteName('enabled') ' = 1')
```

Comment:
@wilsonge we miss a `.` here. :smile: 

Correct would be:
`->set($db->quoteName('enabled') . ' = 1')`